pbe/breakupCoalescenceTables.py

```diff
@@ -0,0 +1,51 @@
+import fluid
+import numpy as np
+import matplotlib.pyplot as plt
+
+path = 'validationData/tables/1/'
+f = open(path + 'parameters', 'w')
+f_br = open(path + 'breakup', 'w')
+f_c = open(path + 'coalescence', 'w')
+
+F = fluid.fluid('coulaloglou', 0, 0)
+F.C1 = 1.52e-02
+F.C2 = 6.78e-02
+F.C3 = 1.06e-12
+F.C4 = 5.13e13
```

Comment:
As you wish, but I don't see the difference between storing them as `C1`, `C2`, `C3` and `C4`. It's not that you're writing `Cbr1`, `Cbr2` and `Ccoal1`, `Ccoal2`. Also you're saving 7-8 lines or code as the definition can be compressed to one line. Note, that naming 4 four variables in the same way and then changing for the index can easily lead to a mistake (e.g. "I forgot to change one index").